paddle/fluid/pybind/inference_api.cc

```diff
@@ -46,6 +47,7 @@ static void BindNativeConfig(py::module *m);
 static void BindNativePredictor(py::module *m);
 static void BindAnalysisConfig(py::module *m);
 static void BindAnalysisPredictor(py::module *m);
+static void BindPaddlePlatform(py::module *m);
```

Comment:
Why not change in `paddle/fluid/pybind/pybind.cc` directly? This function is not used for inference only.

Reply:
ok